did/peer.go

```diff
@@ -258,11 +259,21 @@ func (d DIDPeer) buildVerificationMethod(data, did string) (*VerificationMethod,
 // Extract element purpose and decode each key or service.
 // Insert each key or service into the document according to the designated pu
 func (m PeerMethod2) resolve(did DID, _ ResolutionOptions) (*DIDResolutionResult, error) {
+
 	d, ok := did.(DIDPeer)
 	if !ok {
 		return nil, errors.Wrap(util.CastingError, "did:peer")
 	}
 
+	//if !d.IsValid() {
+	//return nil, errors.New("did is not valid")
+	//}
+
+	ds := string(d)
```

Comment:
can you add a comment here please?

Reply:
done. This had to do with padding. I changed the method during the service decoding section.